Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -171,8 +171,14 @@
 
       <BuildingJob jobTime="0">
       </BuildingJob>
-
-	
+      
+      <Params>
+        <Param name="burnTime" value="0" />
```

Comment:
Should be 4 spaces not 2 in this indent, also looks like there's some tab's in this file